src/log.cpp

```diff
@@ -39,5 +39,10 @@ void librealsense::log_to_file(rs2_log_severity min_severity, const char * file_
 {
 }
 
+void librealsense::log_to_callback(rs2_log_severity min_severity, log_callback_ptr callback)
```

Comment:
Throwing not implemented means that the Viewer wont work. Is that what we want? 
Perhaps log_to methods should just return?

Reply:
@dorodnic The viewer won't be able to get the callbacks, correct -- but we should make sure we catch this error in this case.

I don't know if I like not throwing any error... do you have an alternative? Returning a boolean?

Reply:
I don't think the user should ever call a function that is expected to do something but instead does nothing. Throw an exception, please, or find a way to return some kind of status. Let's talk offline.